newIDE/app/src/Utils/Extension/InstructionGroups.js

```diff
@@ -0,0 +1,54 @@
+import { t } from '@lingui/macro';
+import { type I18n as I18nType } from '@lingui/core';
+
+export const translateInstructionGroup = (
```

Comment:
Could you just do `i18n._(instructionGroup)`? 

I think that will work because:
1. _ handles strings. This both works in the latest version: https://github.com/lingui/js-lingui/blob/04b7cef9876168bc376b55424ecc0b0ebde976c1/packages/core/src/i18n.ts#L211 and in the version we use: https://github.com/lingui/js-lingui/blob/7e2ebbb3b2ec27dc28a7cb661c0daa3455fc3053/packages/core/src/i18n.js#L184, so it's future proof
2. The text to be translated are already in the extension and wrapped inside `_("")` which is extracted into the translation messages catalog, so no need to have the `t` texts explicitly written here?

Reply:
Is Core and the IDE sharing the same translations?

Reply:
Yes!